autoload/vundle/installer.vim

```diff
@@ -357,12 +356,37 @@ endf
 " return -- A 15 character log sha for the current HEAD
 " ---------------------------------------------------------------------------
 func! s:get_current_sha(bundle)
-  let cmd = 'cd '.vundle#installer#shellesc(a:bundle.path()).' && git rev-parse HEAD'
-  let cmd = vundle#installer#shellesc_cd(cmd)
+  let cmd = s:make_sync_command(a:bundle, ['rev-parse', 'HEAD'])
```

Comment:
Hmm... No, of course no. This must be an autocomplete typo.

Le 1 mars 2016 07:18:40 UTC+01:00, Lucas Hoffmann <notifications@github.com> a écrit :
>> @@ -357,12 +356,37 @@ endf
>>  " return -- A 15 character log sha for the current HEAD
>>  "
>---------------------------------------------------------------------------
>>  func! s:get_current_sha(bundle)
>> -  let cmd = 'cd '.vundle#installer#shellesc(a:bundle.path()).' &&
>git rev-parse HEAD'
>> -  let cmd = vundle#installer#shellesc_cd(cmd)
>> +  let cmd = s:make_sync_command(a:bundle, ['rev-parse', 'HEAD'])
>
>Does this work? It looks like the function expects totaly different
>args.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/VundleVim/Vundle.vim/pull/699/files#r54526969